MIT.md

```diff
@@ -1,10 +1,10 @@
-## Copyright 2021, [YOUR NAME]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225813772-2956c728-6dae-4383-b962-dd5da0ef68df.png)

**[OPTIONAL]**

- [ ] You don't have a descriptive pull request summary 😨 Kindly add a descriptive pull request summary to make your pull request descriptive 😊

Reply:
right away 😉